apps/fm/js/fm.js

```diff
@@ -885,9 +885,8 @@ function init() {
 
 document.addEventListener('DOMLocalized', function() {
   // localize MHz
-  document.l10n.format('mhz').then(str => {
-    mhzString = str.value;
-    $('frequency').dataset.unit = mhzString;
+  document.l10n.formatValue('mhz').then(str => {
+    $('frequency').dataset.unit = mhzString = str;
```

Comment:
I see no point in changing anything else than line 888 and 889. At least now it is consistent with the documentation.
You are just making the code less readable.